components/aws/sagemaker/test/integration/README.md

```diff
@@ -0,0 +1,93 @@
+##Requirements
+1. [Conda](https://docs.conda.io/en/latest/miniconda.html)
+1. [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+1. Argo CLI: [Mac](https://github.com/argoproj/homebrew-tap), [Linux](https://eksworkshop.com/advanced/410_batch/install/)
+1. K8s cluster with Kubeflow pipelines > 0.4.0 installed
+1. [IAM Role](https://docs.aws.amazon.com/sagemaker/latest/dg/sagemaker-roles.html) with a SagemakerFullAccess and S3FullAccess
+1. IAM User credentials with SageMakerFullAccess permissions
+
+## Creating S3 buckets with datasets
+Create an S3 bucket and use the following python script to copy `train_data`, `test_data`, and `valid_data.csv` to your buckets.  
+[create the bucket in AWS region where you want to run your tests](https://docs.aws.amazon.com/AmazonS3/latest/gsg/CreatingABucket.html)
+
+Create a new file named `s3_sample_data_creator.py` with following content :
+```python
```

Comment:
Most of it seems to be a README from sample, is it better to just point there ?

Reply:
Sure, that will work for now. We can change it later as we add more tests